dict.js

```diff
@@ -129,12 +129,13 @@ function Dict (defaultValues, opts) {
 
   function listen () {
     Object.keys(sources).forEach(function (key) {
-      releases[key] = bind(sources[key])
+      releases[key] = null
```

Comment:
your code will currently be equivalent to this (as you haven't passed bind a 2nd arg).
Is that inentional ?